resources/test/normalizers/nitoeckyecky.replace

```diff
@@ -0,0 +1 @@
+'ni','ecky ecky ecky'
```

Comment:
Does this file extension mean anything here? (and with the `.regex` file too)

I'd prefer them all to be `.csv`

Reply:
no, extension doesn't mean anything, since it is unit test, I used a number of different extensions